### PR TITLE
Update asm-secret-group-concept.adoc

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

**Review by hegderohit89 — APPROVED** [review]
LGTM

File: modules/ROOT/pages/asm-secret-group-creation-task.adoc [hunk 1]
```diff
@@ -159,7 +159,11 @@ The TLS context you store in Secrets Manager can only be used either when you co
 * *Name* - Enter a name for your TLS context. 
 * *TLS Version* - Select the TLS version. 
 * *Target* - Select the target for the TLS context from the drop-down list. +
-** *Anypoint Security* - Validates the SSL handshake for Runtime Fabric. 
+** *Anypoint Security* - Validates the SSL handshake for Runtime Fabric. +
+If you use the *Anypoint Security* target, you can select keystores and truststores that are either PEM or non-PEM types. Supported types include: +
+* JKS
+* PKCS12
+* JCEKS 
 ** *Mule* - Uses the TLS context as the SSL validation for Mule 4 based API proxies.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-creation-task.adoc:167]: I think you mean for JKS, PKCS12, and JCEKS to be indented under Anypoint Security. If so, change the * to *** for all three.

File: modules/ROOT/pages/asm-secret-group-concept.adoc [hunk 2]
```diff
@@ -5,19 +5,19 @@ endif::[]
 
 A secret group is a unique secrets vault to which all your secrets are bound. Each secret group has individual access and authorization controls.
 
-Secrets Manager stores your secrets per secret group, per environment and per business group. Only applications deployed to those specific business groups and environments within your organization can access the secrets stored in each secret group.
+Secrets Manager stores your secrets per secret group, per environment and per business group. User access and authorization controls are applied at the environment level. A user who has access on a given environment can access all the secret groups that belong to that environment. Only applications deployed to those specific business groups and environments within your organization can access the secrets stored in each secret group.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/asm-secret-group-concept.adoc:8]: Great way to clarify the diff between what a user sees and what the app "sees."

**Review by Mulesoft-Mysti — APPROVED** [review]
Just one indent issue (unless I'm confused). Great!

**Pull request merged**